src/modeling/models/mmt_encoder.py

```diff
@@ -203,3 +242,35 @@ def get_embedding_table(self):
           'Cannot call `get_token_embedding_table()` until the model has been '
           'called so that all variables are built.')
     return self._word_embedding_layer.embedding_table
+
+  def get_embedding_layer(self):
```

Comment:
"get"embedding_layer" is ambiguous, right? it could mean the "word embedding" layer or the "image patch embedding" layer. I'd add "word" to the name here.

Reply:
Yeah! It makes sense to add "word" here. Thanks!